helm-charts/sawmills-collector/tests/resource_base_name_test.yaml

```
@@ -0,0 +1,254 @@
+      loadBalancer.enabled: true
+      loadBalancer.service.headless.enabled: true
+      haproxy.mapping:
+        http_4318:
+          from: 4318
+          to:
+            port: 4318
+            fallback_endpoint: vendor.example.com
+    asserts:
+      - matchRegex:
+          path: data["haproxy.cfg"]
+          pattern: "server-template sibling 10 coinbase-logs-headless\\.customer-ns\\.svc\\.cluster\\.local:4318.*check port 13136.*backup.*resolvers k8s.*init-addr none"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)